charts/rstudio-workbench/templates/_helpers.tpl

```diff
@@ -306,10 +316,54 @@ volumes:
     name: {{ include "rstudio-workbench.fullname" . }}-pam
     defaultMode: {{ .Values.config.defaultMode.pam }}
 {{- end }}
+{{- if or .Values.launcherPem.existingSecret .Values.secureCookieKey.existingSecret .Values.config.database.conf.existingSecret .Values.config.secret }}
 - name: rstudio-secret
-  secret:
-    secretName: {{ include "rstudio-workbench.fullname" . }}-secret
-    defaultMode: {{ .Values.config.defaultMode.secret }}
+  projected:
+    sources:
+{{- if .Values.config.secret }}
+{{- range $key, $value := .Values.config.secret }}
+    - secret:
+      name: {{ $key }}
+      items:
+      - key: {{ $key }}
+        value: {{ $value }}
+        path: {{ $key }}
+        mode: {{ $.Values.config.defaultMode.secret }}
+{{- end }}
+{{- end }}
+{{- if .Values.launcherPem.existingSecret  }}
+    - secret:
+      name: launcher-pem-secret
+      items:
+      - key: {{ .Values.launcherPem.existingSecret }}
+        path: launcher.pem
+        mode: {{ .Values.config.defaultMode.secret }}
+{{- end }}
```

Comment:
Thanks for catching the underlying issue here @jforest!

Given the complexity and importance of this, could we please add a `secrets_test.yml` to https://github.com/rstudio/helm/tree/main/charts/rstudio-workbench/tests to help catch regressions?

Please let me know how I can help with this.